routes/celebrities.routes.js

```diff
@@ -0,0 +1,41 @@
+// starter code in both routes/celebrities.routes.js and routes/movies.routes.js
+const router = require("express").Router();
+
+// all your routes here
+const app = require("../app");
+
+const Celebrity = require('../models/Celebrity.model.js');
+
+
+router.get('/celebrities/create', (req, res, next) => {
+    
+    res.render('celebrities/new-celebrity');
+})
```

Comment:
- when I got to page to create celebrity, I get below error in terminal:
```
Error: Failed to lookup view "celebrities/new-celebrity" in views directory
```

- You are rendering this: `res.render('celebrities/new-celebrity');`
-  Your file is called `new-celebrities.hbs`